FRAuth/FRAuthTests/FRAuthSwiftTests/E2ETests/Callback-Live/AA-05-DeviceBindingCallbackTest.swift

```diff
@@ -39,6 +39,18 @@ class AA_05_DeviceBindingCallbackTest: CallbackBaseTest {
     }
     
     override func tearDown() {
+        let frUserKeys = FRUserKeys()
+        var userKeys: [UserKey] = []
+        userKeys = frUserKeys.loadAll()
+        
+        for (_, userKey) in userKeys.enumerated().reversed()
```

Comment:
any reason we are doing reversed?  Also let's open the { on the same line as the for statement for consistency

Reply:
Yes, I copied this code from the FRExample app... 
Fixing this...